en-US/mozorg/about/governance/policies/participation.lang

```diff
@@ -386,5 +386,13 @@ This set of guidelines is distributed under a <a href="%(license)s" rel="license
 These guidelines have been adapted with modifications from Mozilla’s original Community Participation Guidelines, the <a href="%(ubuntu_coc)s">Ubuntu Code of Conduct</a>, Mozilla’s <a href="%(viewsource_coc)s">View Source Conference Code of Conduct</a>, and the <a href="%(rustlang_coc)s">Rust Language Code of Conduct</a>, which are based on Stumptown Syndicate’s <a href="%(citizen_coc)s">Citizen Code of Conduct</a>. Additional text from the <a href="%(lgbtqtech_coc)s">LGBTQ in Technology Code of Conduct</a> and the <a href="%(wiscon_coc)s">WisCon code of conduct</a>. This document and all associated processes are only possible with the hard work of many, many Mozillians.
 
 
+;Modifications to these guidelines
+Modifications to these guidelines
+
+
+;Mozilla may amend the guidelines from time to time and may also vary the procedures it sets out where appropriate in a particular case. Your agreement to comply with the guidelines will be deemed agreement to any changes to it. In case Mozilla amends the guidelines, the updated version can be found at <a href="{{ guidelines }}">https://www.mozilla.org/about/governance/policies/participation/</a>. You can sign up for push updates by subscribing to <a href="{{ group }}">https://groups.google.com/d/forum/mozilla-cpg</a>. This policy does not form part of any Mozilla employee’s contract of employment or otherwise have contractual effect.
+Mozilla may amend the guidelines from time to time and may also vary the procedures it sets out where appropriate in a particular case. Your agreement to comply with the guidelines will be deemed agreement to any changes to it. In case Mozilla amends the guidelines, the updated version can be found at <a href="{{ guidelines }}">https://www.mozilla.org/about/governance/policies/participation/</a>. You can sign up for push updates by subscribing to <a href="{{ group }}">https://groups.google.com/d/forum/mozilla-cpg</a>. This policy does not form part of any Mozilla employee’s contract of employment or otherwise have contractual effect.
```

Comment:
so the change is from <a href="{{ group }}"> to <a href="%(guidelines)s">, correct?